ci/scripts/wait-for-deploy.js

```diff
@@ -40,13 +41,8 @@ const run = async function run() {
         clearInterval(deploymentChecker)
       }
       if (rolloutState === 'IN_PROGRESS' && counter === 1) {
-        console.log(
-          `Deploying task definition: ${taskDefinition}
-           Current status: ${rolloutState}
-           Desired count: ${desiredCount}
-           Pending count: ${pendingCount}
-           Running count: ${runningCount}`
-        )
+        console.log('Deployment details:')
+        console.table({taskDefinition, deploymentStatus: rolloutState, appVersion})
```

Comment:
I like this idea. Looking at the latest toolbox wait for deploy task it doesn't look like it printed the `appVersion`?
https://cd.gds-reliability.engineering/teams/pay-dev/pipelines/apps-test-ecr/jobs/deploy-toolbox/builds/109#L6033a1ac:8

Reply:
probs hadn't set pipeline to pass the tag through